### PR TITLE
Gitignore & FB integration with proj setup

### PR DESCRIPTION
**Pull request by vineet-webonise**

### CODE REVIEW

File: SUNFLOWR/SUNFLOWR/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,84 @@
+//
+//  AppDelegate.swift
+//  SUNFLOWR
```
Comment (saketPandhare-Webonise) [inline SUNFLOWR/SUNFLOWR/AppDelegate.swift:3]: Remove Below lines 5,6,7

File: SUNFLOWR/SUNFLOWR/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        // Use this method to select a configuration to create the new scene with.
+        return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
+    }
+
+    func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
+        // Called when the user discards a scene session.
+        // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
+        // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
+    }
+
```
Comment (saketPandhare-Webonise) [inline SUNFLOWR/SUNFLOWR/AppDelegate.swift:37]: Does our app Requires Usage of CoreData ?

File: SUNFLOWR/SUNFLOWR/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,20 @@
+//
+//  ViewController.swift
+//  SUNFLOWR
+//
```
Comment (saketPandhare-Webonise) [inline SUNFLOWR/SUNFLOWR/ViewController.swift:4]: Remove below lines 5,6and 7

File: SUNFLOWR/SUNFLOWR/AppDelegate.swift (hunk 2 above)
Comment (vineet-webonise) [reply SUNFLOWR/SUNFLOWR/AppDelegate.swift:37]: As per the design we do require local database, its yet to decide to use CoreData.

File: SUNFLOWR/SUNFLOWR/AppDelegate.swift (hunk 1 above)
Comment (vineet-webonise) [reply SUNFLOWR/SUNFLOWR/AppDelegate.swift:3]: Done

File: SUNFLOWR/SUNFLOWR/ViewController.swift (hunk 3 above)
Comment (vineet-webonise) [reply SUNFLOWR/SUNFLOWR/ViewController.swift:4]: Done

File: SUNFLOWR/SUNFLOWR/AppDelegate.swift (hunk 2 above)
Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/AppDelegate.swift:37]: ok

**Pull request merged**